packages/repo-tools/src/lib/runner.ts

```diff
@@ -37,6 +37,7 @@ export async function runner(
       port: options.startingPort,
       stopPort: options.startingPort + 1_000,
     }));
+  console.log(packages);
```

Comment:
did you intend to add this here or was it for debugging purposes only?

Reply:
😅 , debugging, will remove.